### PR TITLE
feat - Added rating and grid components

### PR DESCRIPTION
**Pull request by akash-1318**
https://user-images.githubusercontent.com/64393554/153645431-bb9705b1-9546-4afd-b8bb-93ab168dd51b.mov

Added rating and grid-simplified components.

### CODE REVIEW

File: components/Grid.html [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Milkyway</title>
+    <link rel="stylesheet" href="css/main.css">
+    </link>
+    <link href='https://unpkg.com/boxicons@2.1.1/css/boxicons.min.css' rel='stylesheet'>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto+Slab:wght@300;400;500&display=swap" rel="stylesheet">
```
Comment (darshandamre) [inline components/Grid.html:12]: link should be auto closing tags

File: components/Grid.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+            <div class="main__grid-item">
+                <div class="component">
+                    <p class="component__heading">Grid</p>
+                    <p class="component__text">Grid is used to create layout of your webpage. Grid comes 
+                        in really handy dealing with complex layout and also helps in improving 
+                        responsive webpages.</p>
+                    <p class="component__heading-primary">Grid with two Items</p>
+                    <p class="component__sub-text">
+
+                    </p>
```
Comment (darshandamre) [inline components/Grid.html]: empty tags

File: components/Grid.html (hunk 1 above)
Comment (darshandamre) [reply components/Grid.html:12]: same in other file

File: components/Rating.html [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+                    <div class="code__snippet">
+                        <iframe
+                        src="https://carbon.now.sh/embed/U7LMvQCzdsFGwnEJneBy"
+                        sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
+
+                    <p class="component__heading-primary">Disabled ratings</p>
+                    <p class="component__sub-text">
+                        For using disabled rating add <span class="comp__detail-text">.disabale_rating</span> class.
```
Comment (irajbhinde) [inline components/Rating.html:110]: can use a better class name here

File: components/css/componentStyle/Rating.css [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+.ratings.md > i{
+    font-size: 2rem;
+}
+
+.ratings.sm > i{
+    font-size: 1.5rem;
+}
+
+.ratings > :nth-child(1),.ratings > :nth-child(2),.ratings > :nth-child(3){
```
Comment (irajbhinde) [inline components/css/componentStyle/Rating.css:17]: Liked how you used nth child to fill colors to the icons. Great work here

**Review by irajbhinde — COMMENTED** [review]
Looks fine. In the future when you get time, you can increase the width & height of pre codes for web page to look better

**Pull request merged**